ui/push-health/GroupedTests.jsx

```diff
@@ -34,6 +48,56 @@ class GroupedTests extends PureComponent {
     });
   };
 
+  addToCheckJob = (job) => {
+    this.setState((prevState) => ({
+      checkedJobs: [...prevState.checkedJobs, job],
+    }));
+  };
+
+  removeFromCheckedJobs = (uncheckedJob) => {
+    const index = this.state.checkedJobs.findIndex((job) => {
+      return uncheckedJob === job;
+    });
+    const { checkedJobs } = this.state;
+    checkedJobs.splice(index, 1);
+    this.setState(checkedJobs);
+  };
+
+  retriggerSelected = (times, group) => {
+    const { notify, currentRepo, revision } = this.props;
+    // Reduce down to the unique jobs
+    const jobs = group.tests.reduce((acc, test) => {
+      if (
+        this.state.checkedJobs.findIndex((job) => {
```

Comment:
instead of referencing `this.state.checkedJobs` we prefer using deconstruction: `const { checkedJobs } = this.state`